packages/components/src/components/Snackbar/useSnackbar.tsx

```diff
@@ -20,11 +20,9 @@ export function useSnackbar() {
     throw new Error('useSnackbar must be used within a SnackbarProvider')
   }
 
-  const show = (
-    message: string,
-    snackbarOptions?: modifyToastOptions['data'],
-  ) => {
-    const { offset, setOffset } = context
+  const { offset, setOffset } = context
+
+  const show = (message: string, snackbarOptions?: SnackbarOptions) => {
```

Comment:
Doesn't matter, but just noting that `const { offset, setOffset } = context` was moved out of `show`. Not sure if intentional or just a minor inconsistency fixing conflicts with my changes.

Reply:
Will be cleaned up further in the screenreader dismiss ticket #440